cdap-ui/app/cdap/components/AbstractWidget/KeyValueDropdownWidget/KeyValueDropdownRow.tsx

```diff
@@ -131,14 +143,34 @@ class KeyValueDropdownRow extends AbstractRow<IKeyValueDropdownRowProps, IKeyVal
           disabled={this.props.disabled}
           data-cy="value"
         >
-          {dropdownOptions.map((option) => {
-            return (
+          {data.map((option, index) =>
```

Comment:
nit: IMO we shouldn't do this rename. The dropdownOptions shows that it is a list of options (array). It is hard to infer the type from the name `data`.